Bottom: 228735efe62c4ad179b683d4a4d94f2127cc5169
Top:    e07f7b3a50f369215e5831da1ed0562824f8b1f1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:56:11 +0000

Turn off debug hacks in object generation


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c8a0d8e..f79622e 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1234,9 +1234,9 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   destroy_appearance (copper_appearance);
 
   /* DEBUG */
-  poly_M_Copy0 (&PCB->Data->outline, info.poly);
-  PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  poly_M_Copy0 (&PCB->Data->outline, info.poly);
+//  PCB->Data->outline_valid = true;
+//  gui->invalidate_all ();
 
   poly_Free (&info.poly);
